Marlin/language_cz.h

```diff
@@ -77,6 +78,68 @@
 #define MSG_MOVE_AXIS                       _UxGT("Posunout osy")
 #define MSG_BED_LEVELING                    _UxGT("Vyrovnat podlozku")
 #define MSG_LEVEL_BED                       _UxGT("Vyrovnat podlozku")
+#define MSG_EDITING_STOPPED                 _UxGT("Konec uprav site")
+
+#define MSG_UBL_DOING_G29                   _UxGT("Provadim G29")
+#define MSG_UBL_UNHOMED                     _UxGT("Prejedte domu")
+#define MSG_UBL_TOOLS                       _UxGT("UBL nastroje")
+#define MSG_UBL_LEVEL_BED                   _UxGT("Unified Bed Leveling")
+#define MSG_UBL_MANUAL_MESH                 _UxGT("Manualni sit bodu")
+#define MSG_UBL_BC_INSERT                   _UxGT("Polozte kreditku a zmerte")
```

Comment:
You can test for over-long strings using a "ruler". These strings are all too long:
```
12345678901234567890
Polozte kreditku a zmerte
Odstrnte a zmerte podlozku
Zkontrolovat vlastni sit
```